src/Services/Terminal/TerminalParser.js

```diff
@@ -0,0 +1,48 @@
+import {
+  TerminalRuntimeError,
+  TerminalParsingError,
+} from './TerminalErrors';
+
+const errorHandler = (rsp) => {
+  console.log(rsp);
+  throw new TerminalRuntimeError();
+};
+
+const createSession = (rsp) => {
+  if (
+    !rsp['common_v33_0:HostToken'] ||
```

Comment:
Since uapi_version is auto-detecting from response I think it's better to use `this.uapi_version` here to support future updates of uapi version